include/drives.h

```diff
@@ -94,6 +97,7 @@ class localDrive : public DOS_Drive {
 
 private:
 	bool IsFirstEncounter(const std::string& filename);
+	bool always_allow_rw_open;
```

Comment:
Suggest aligning this variable with the conf name, maybe `bool always_open_protected_files = `?

Reply:
Way too long + current name is more descriptive of the actual internal functionality.

Reply:
`always_open_ro_files`